setup.cfg

```diff
@@ -1,26 +1,56 @@
 [metadata]
 name = grizli
-description = Grism analysis pipeline
+version = 1.6.0
 author = G. Brammer
 author_email = gbrammer@gmail.com
+description = Grism redshift and line analysis software
+long_description = file: README.rst
+url = https://github.com/gbrammer/grizli
 license = MIT
-url = http://github.com/gbrammer/grizli
-edit_on_github = False
-github_project = gbrammer/grizli
+license_files = LICENSE.txt
+project_urls = 
+    Documentation = https://grizli.readthedocs.io/
+    Source = https://github.com/astropy/astropy
+classifiers=
+        Development Status :: 1 - Planning
+        Intended Audience :: Science/Research
+        Topic :: Scientific/Engineering :: Astronomy
+
+# Options specific to Astropy-affiliated packages
+# See below for more info :
+# https://docs.astropy.org/projects/package-template/en/latest/options.html
+# edit_on_github_extension = False
+# github_project = gbrammer/grizli
+
+[options]
+python_requires = >=3.7
 install_requires = 
-    numpy
-    cython
     astropy
+    cython
+    numpy
     sregion>=1.0
-    
-description-file = README.rst
-python_requires = ">=3.7"
-version = 1.6.0
+packages = find_namespace:
+package_dir = 
+    =grizli
```

Comment:
I think this is only needed if the package dir is different from the package name.  Since they are the same, it is redundant.

Reply:
Ah, alright. Will remove and push once more.